godel_surface_detection/src/nodes/robot_auto_blending.cpp

```diff
@@ -23,62 +23,66 @@ const static int MAX_SERVICECALL_FAILURE = 20;
 int main(int argc, char** argv)
 {
   ros::init(argc, argv, "robot_auto_blending");
-  ros::NodeHandle nh;
+  ros::NodeHandle node_handle;
   ros::Rate loop_rate(10);
 
   godel_msgs::SurfaceBlendingParameters param_srv;
-  godel_msgs::SurfaceDetection surfacedetection_srv;
-  godel_msgs::SelectSurface::Request req;
-  godel_msgs::SelectSurface::Response res;
+  godel_msgs::SurfaceDetection surfaceDetection_srv;
```

Comment:
Variable names need to be [**under_scored**](http://wiki.ros.org/CppStyleGuide#Naming), for instance the variable "surfaceDetection_srv" should be **surface_detection_srv**